client/src/components/Donation/DonateForm.tsx

```diff
@@ -141,17 +186,17 @@ class DonateForm extends Component {
     return donationBtnLabel;
   }
 
-  handleSelectDuration(donationDuration) {
+  handleSelectDuration(donationDuration: 'month' | 'onetime') {
     const donationAmount = this.getActiveDonationAmount(donationDuration, 0);
     this.setState({ donationDuration, donationAmount });
   }
 
-  handleSelectAmount(donationAmount) {
+  handleSelectAmount(donationAmount: number) {
     this.setState({ donationAmount });
   }
 
-  renderAmountButtons(duration) {
-    return this.amounts[duration].map(amount => (
+  renderAmountButtons(duration: 'month' | 'onetime') {
```

Comment:
GitHub won't let me leave a comment in the right place :facepalm:, but 

```
(Object.keys(this.durations) as ['month' | 'onetime'])
```

needs to replace line 243 (just before `<Tab`)